### ISSUE TITLE
Use of uninitialized value $check_postgres::opt{"output"} 

### ISSUE THREAD

**Issue by mdeguzis** [bucardo/check_postgres#141]
Check does seem to go ok:

```
Use of uninitialized value $check_postgres::opt{"output"} in lc at /usr/lib64/nagios/plugins/postgres/check_postgres_connection line 711.
POSTGRES_CONNECTION OK: DB "icinga" (host:icinga-postgres-01) version 9.2.2
```

command used:
```
postgres/check_postgres_connection -db icinga -u icinga -H icinga-postgres-01
```

Deployed using symlink install option

**Comment by machack666** [comment]
Which version of check_postgres is this?

**Comment by mdeguzis** [comment]
The latest tagged release

```
2.23.0
```

**Comment by turnstep** [comment]
Fixed in cf2dc459856ce17689ecc816ce1ec04e8ff93e91
